src/commands.ts

```diff
@@ -0,0 +1,33 @@
+import { Options, Scope } from "./validate";
+
+export function getAddCommand(options: Options): string {
+    let command: string;
+
+    if (options.SCOPE === Scope.SiteCollection) {
+        command = `spo app add --filePath ${options.APP_FILE_PATH} --appCatalogScope ${options.SCOPE} --appCatalogUrl ${options.SITE_COLLECTION_URL}`;
+    } else {
+        command = `spo app add --filePath ${options.APP_FILE_PATH}`;
+    }
+
+    if (options.OVERWRITE) {
+        command += " --overwrite";
```

Comment:
I would prefer to use single quotes here so we are consistent with the test file.

```suggestion
        command += ' --overwrite';
```